releasenotes/notes/add-graph6-support.yaml

```diff
@@ -0,0 +1,48 @@
+features:
```

Comment:
Please read https://docs.openstack.org/reno/latest/user/usage.html#editing-a-release-note for the correct format of release notes